src/test/groovy/nebula/plugin/publishing/publications/SourceJarPluginIntegrationSpec.groovy

```diff
@@ -122,7 +122,7 @@ class SourceJarPluginIntegrationSpec extends IntegrationSpec {
         def ivyXml = new XmlSlurper().parse(ivyXmlFile)
 
         then:
-        ivyXml.publications[0].artifact.find { it.@type == 'sources' && it.@conf == 'sources' }
+        ivyXml.publications[0].artifact.find { it.@type == 'jar' && it.@conf == 'sourcesElements' }
```

Comment:
I'm wondering if this change causes any complications? Any thoughts? @rpalcolea, @OdysseusLives, @DanielThomas The classifier is still the same but switching to gradle variant to add source jar impacted `type` and `configuration` properties in ivy metadata.